src/Views/ExpenseForm.elm

```diff
@@ -0,0 +1,55 @@
+module Views.ExpenseForm exposing (view)
+
+import Date
+import Html exposing (Html, text, div, img)
+import Material.Button as Button
+import Material.Options as Options
+import Material.Textfield as Textfield
+import Model exposing (Model, ExpenseType(Debit))
+import Msg exposing (..)
+
+
+isFloat : String -> Bool
+isFloat n =
+    case String.toFloat n of
+        Ok _ ->
+            True
+
+        Err _ ->
+            False
+
+
+view : Model -> Html Msg
```

Comment:
I would like to write

```
view : ExpenseForm -> Html Msg
```

but I can't because I need to pass material model to the fields. Now, I understand why the component model is not cool. If I don't specify `Textfield.value` then material component will use their intern data to persist what user typed so far.